.github/workflows/ci.yml

```diff
@@ -59,6 +59,45 @@ jobs:
           echo "NIXPKGS_SYSTEM=$(nix eval --raw 'nixpkgs#system')" >> "$GITHUB_ENV"
       - run: nix build -L ".#devShells.${NIXPKGS_SYSTEM}.default" --keep-going
 
+  nix-flake-check:
```

Comment:
The equivalent of `nix flake check` is already done with nix-github-actions, so that can be removed.